.script/tests/yamlFileValidatorTest/yamlFileValidator.test.ts

```diff
@@ -0,0 +1,21 @@
+import { IsValidYamlFile } from "../../yamlFileValidator";
+import { ExitCode } from '../../utils/exitCode';
+import chai from 'chai';
```

Comment:
chai - is this a known trust-able lib?

Reply:
yes. I see other repo in msazure/one project that use this library